http/src/main/java/org/http4s/blaze/http/parser/Http1ServerParser.java

```diff
@@ -48,6 +48,10 @@ public final boolean requestLineComplete() {
         return _lineState == LineState.END;
     }
 
+    public final boolean isReset() {
```

Comment:
I don't think I'd think something `isReset` if it's never been used.  `isStart`, maybe?